snap/snapcraft.yaml

```diff
@@ -45,10 +47,11 @@ parts:
     prime:
       - -usr/include
     organize:
-      ./usr/share/srslte/ue.conf.example: ./usr/share/srslte/ue.conf
-      ./usr/share/srslte/drb.conf.example: ./usr/share/srslte/drb.conf
-      ./usr/share/srslte/enb.conf.example: ./usr/share/srslte/enb.conf
-      ./usr/share/srslte/epc.conf.example: ./usr/share/srslte/epc.conf
-      ./usr/share/srslte/rr.conf.example: ./usr/share/srslte/rr.conf
-      ./usr/share/srslte/sib.conf.example: ./usr/share/srslte/sib.conf
-      ./usr/share/srslte/user_db.csv.example: ./usr/share/srslte/user_db.csv
+      ./usr/share/srsran/ue.conf.example: ./usr/share/srsran/ue.conf
+      ./usr/share/srsran/drb.conf.example: ./usr/share/srsran/drb.conf
+      ./usr/share/srsran/enb.conf.example: ./usr/share/srsran/enb.conf
+      ./usr/share/srsran/epc.conf.example: ./usr/share/srsran/epc.conf
+      ./usr/share/srsran/rr.conf.example: ./usr/share/srsran/rr.conf
+      ./usr/share/srsran/sib.conf.example: ./usr/share/srsran/sib.conf
+      ./usr/share/srsran/user_db.csv.example: ./usr/share/srsran/user_db.csv
+      ./usr/share/srsran/rb.conf.example: ./usr/share/srsran/rb.conf
```

Comment:
Did we intentionally remove `rb.conf.example`?

Reply:
in the 20_10_1 version you mean? it wasn't present among the example config files.
now in 22_10 it is, and srsenb would crash if it doesn't find it.

Reply:
Understood, resolved.